src/main.ts

```diff
@@ -59,39 +63,46 @@ export async function run(): Promise<void> {
   }
 }
 
-const attest = async (
+const createAttestation = async (
   subject: Subject,
   visibility: Visibility
 ): Promise<Attestation> => {
-  const provenance = generateProvenance(subject, process.env)
-
+  const predicate = await buildSLSAProvenancePredicate()
   core.startGroup(
     highlight(
       `Provenance attestation generated for ${subject.name} (sha256:${subject.digest.sha256})`
     )
   )
-  core.info(JSON.stringify(provenance, null, 2))
+  core.info(
```

Comment:
What does the info function do? Does it print content to the console during an Actions run?

Reply:
Yeah, the info function allows us to write to the job ouput:
<img width="657" alt="image" src="https://github.com/github-early-access/generate-build-provenance/assets/398027/e1bf3dcf-4e46-44a4-a96a-abe6b78f236f">